programs/eosioc/main.cpp

```diff
@@ -682,8 +682,14 @@ int main( int argc, char** argv ) {
       std::string wast;
       std::cout << localized("Reading WAST...") << std::endl;
       fc::read_file_contents(wastPath, wast);
-      std::cout << localized("Assembling WASM...") << std::endl;
-      auto wasm = assemble_wast(wast);
+
+      vector<uint8_t> wasm;
+      if(wast.compare(0, 4, "\x00\x61\x73\x6d"))
```

Comment:
Can we pull `"\x00\x61\x73\x6d"` into something more descriptively named (maybe: `const char wasm_magic_bytes[4]`)  and put a comment on it that points to: 
https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md#high-level-structure